src/Components/LeftSideWeatherApp/CurrentDayWeatherForecast/CurrentDayWeatherForecast.js

```diff
@@ -1,23 +1,24 @@
 import './CurrentDayWeatherForecast.css';
                                   
 function CurrentDayWeatherForecast(props) {
-    const weekday = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+    const weekday = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
     const d = new Date();
```

Comment:
give meaningful name to variables, i should have an idea what a variable means by just looking at its name. This can be `currentDate` for example